src/CattyTests/Firebase/FirebaseCrashlyticsReporterTests.swift

```diff
@@ -21,54 +21,60 @@
  */
 
 import Firebase
+
 @testable import Pocket_Code
+
 import XCTest
 
-final class FirebaseCrashlyticsSetupTests: XCTestCase {
+final class FirebaseCrashlyticsReporterTests: XCTestCase {
 
-    var app: AppDelegate?
     var crashlytics: CrashlyticsMock?
+    var reporter: FirebaseCrashlyticsReporter?
 
     override func setUp() {
         super.setUp()
 
         crashlytics = CrashlyticsMock(collectionEnabled: false)
-        app = AppDelegateMock(crashlytics: crashlytics!)
+        reporter = FirebaseCrashlyticsReporter(crashlytics: crashlytics!)
+    }
+
+    func testAppDelegateInitialization() {
+        let app = AppDelegate()
+
+        XCTAssertNil(app.firebaseAnalyticsReporter)
+        XCTAssertNil(app.firebaseCrashlyticsReporter)
     }
 
     func testSetupCrashReportsEnabled() {
         UserDefaults.standard.set(true, forKey: kFirebaseSendCrashReports)
 
-        app?.setupCrashlytics()
+        _ = FirebaseCrashlyticsReporter(crashlytics: crashlytics!)
         XCTAssertTrue(crashlytics!.isCrashlyticsCollectionEnabled())
     }
 
     func testSetupCrashReportsDisabled() {
         UserDefaults.standard.set(false, forKey: kFirebaseSendCrashReports)
 
-        app?.setupCrashlytics()
+        _ = FirebaseCrashlyticsReporter(crashlytics: crashlytics!)
         XCTAssertFalse(crashlytics!.isCrashlyticsCollectionEnabled())
     }
 
     func testSettingsDisabledNotification() {
         UserDefaults.standard.set(true, forKey: kFirebaseSendCrashReports)
```

Comment:
My suggestion would be to move the default setting (true) to the `setUp` method.